frappe/public/js/frappe-web.bundle.js

```diff
@@ -24,3 +24,4 @@ import "./bootstrap-4-web.bundle";
 
 import "../../website/js/website.js";
 import "./frappe/socketio_client.js";
+import "./frappe/ui/chart.js";
```

Comment:
All this file does is load Frappe Charts (and a realtime wrapper around it). This can be loaded asynchronously too. 

Not sure why this should be the default website bundle? Websites don't really need charts that often 😅